examples/basics/batches.ipynb

```diff
@@ -1,354 +1,434 @@
 {
```

Comment:
<p>Are they in beta? I think we want everyone to use them because we are trying to transition away from datasets. This might scare people away from using them. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/750/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Not anymore I think.  I will remove the beta from the title.